man/osg-incommon-cert-request.1

```diff
@@ -1,6 +1,7 @@
 .TH osg-incommon-cert-request 1
 .SH NAME
-osg-incommon-cert-request \- retrieves host and service certificates from the InCommon CA
+osg-incommon-cert-request \- retrieves host and service certificates from the \\
```

Comment:
The extra double-backslash here doesn't help, it just ends up with a slash in the middle of the output text.  They're only helpful in the `.nf` sections that have formatting disabled.

Reply:
Removed